src/crypto/slow-hash.c

```diff
@@ -1070,6 +1069,13 @@ void cn_slow_hash(const void *data, size_t length, char *hash)
         hash_extra_blake, hash_extra_groestl, hash_extra_jh, hash_extra_skein
     };
 
+#ifndef FORCE_USE_HEAP
+    uint8_t long_state[MEMORY];
+#else
+    uint8_t *long_state = NULL;
+    long_state = (uint8_t *)malloc(MEMORY);
+#endif
```

Comment:
From what i see, there is no loop calling cn_slow_hash in the wallet and i don't think that this define will be necessary for the daemon, since a better fix would be to set stack size during the thread creation, and the cli works fine without